bot-components/GitHub_queries.ml

```diff
@@ -131,17 +93,16 @@ let pull_request_milestone_and_cards ~bot_info ~owner ~repo ~number =
   | Error err ->
       Error err
 
-type mv_card_to_column_input = {card_id: string; column_id: string}
-
-let backported_pr_info ~bot_info number base_ref =
-  pull_request_milestone_and_cards ~bot_info ~owner:"coq" ~repo:"coq" ~number
-  >|= function
+let get_backported_pr_info ~bot_info number base_ref =
+  get_pull_request_milestone_and_cards ~bot_info ~owner:"coq" ~repo:"coq"
+    ~number
+  >>= function
```

Comment:
Instead of a `>>=` here and a `Lwt.return` in each branch, you can probably use a `>|=` here.